docs/_tutorials/convert-existing-site-to-jekyll.md

```diff
@@ -1,39 +1,44 @@
 ---
 layout: tutorials
-permalink: /tutorials/convert-existing-html-sites-to-jekyll-templates/
-title: Convert existing HTML sites to Jekyll Templates
+permalink: /tutorials/convert-site-to-jekyll/
+title: Convert an HTML site to Jekyll
 ---
 
-If you're looking for themes for your Jekyll site, you don't have to restrict yourself to existing Jekyll themes. It's pretty easy to convert almost any static HTML site into a Jekyll theme.
+If you're looking for themes for your Jekyll site, you don't have to restrict yourself to existing Jekyll themes. It's pretty easy to convert almost any static HTML files into a Jekyll website.
 
-In many ways, any site that is currently a static site is *already* a Jekyll site. Jekyll just allows you to automate parts of the site (like inserting pages into templates, rendering lists for navigation, generating feeds and sitemaps, and more) as it processes the files.
+In many ways, any site that is currently a static site is *already* a Jekyll website. Jekyll just allows you to automate parts of the site (like inserting pages into templates, rendering lists for navigation, generating feeds and sitemaps, and more) as it processes the files.
 
-Understanding how to convert any HTML site into a Jekyll template will open your world to many more options for Jekyll themes. Instead of searching online for "Jekyll themes," you can choose from the large variety of HTML templates for your site, quickly Jekyll-ize the HTML template as you need to, and build the output with Jekyll.
+Understanding how to convert any HTML site into Jekyll templates will open your world to many more options for Jekyll themes. Instead of [searching online for  *Jekyll themes*](https://duckduckgo.com/?q=Jekyll+themes), you can choose from the large variety of HTML templates for your site, quickly Jekyll-ize the HTML templates as you need to, and build the output with Jekyll.
 
-Although websites can have sophisticated features and controls, we'll keep things simple in this tutorial. This tutorial contains the following sections:
+Although websites can have sophisticated features and controls, we'll keep things simple in this tutorial.
 
-* TOC
-{:toc}
-
-## Understanding a basic Jekyll site
+## What is a Jekyll Website?
 
 First, let's start with a grounding in the basics. Stripping a Jekyll site down to an extremely basic level will help clarify what happens in a Jekyll site. If you haven't already installed the jekyll gem, [install it]({% link _docs/installation.md %}).
 
-A bare boned Jekyll Site consists of:
-
-  * an empty `_config.yml`
-  * an `index.html` containing the entire source-code for your site's homepage.
-
-But for this tutorial, we'll start with a *basic Jekyll site* consisting of 3 files:
+A bare boned Jekyll site consists of a bunch of HTML, CSS — or Sass — and JavaScript files. In this tutorial, we'll start with a *basic Jekyll site* consisting of three files:
```

Comment:
> A bare boned Jekyll site consists of a bunch of HTML, CSS — or Sass — and JavaScript files

It wouldn't be bare boned when there are bunch of HTML, CSS, and other asset files. Bare bone sites have the most minimum no. of files